### PR TITLE
refactor(VGMdb): change `.querySelector`s

### PR DESCRIPTION
**Pull request by kyrie25**
**Describe the changes in this pull request:**
Refactored some `.querySelector`s because `:nth-child()` is unreliable and has broke one of the `.querySelector` itself
<details>
<summary> Proof showing the creation/modification is working as expected </summary>
<br>

image
image

</details>

### CODE REVIEW

File: websites/V/VGMdb/dist/metadata.json [hunk 1]
```diff
@@ -10,18 +10,12 @@
...
 	"color": "#B90000",
 	"category": "music",
-	"tags": [
-		"video",
-		"game",
-		"music",
-		"database",
-		"album"
-	],
+	"tags": ["video", "game", "music", "database", "album"],
```
Comment (EncryptedDev) [inline websites/V/VGMdb/dist/metadata.json:18]: Revert this change, Prettier prefers the former version

**Review by EncryptedDev — DISMISSED** [review]

File: websites/V/VGMdb/dist/metadata.json (hunk 1 above)
Comment (kyrie25) [reply websites/V/VGMdb/dist/metadata.json:18]: Not sure because this is actually Prettier's doing, but fixed reverted nonetheless

**Review by Slowlife01 — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]
